.github/pull_request_template.md

```diff
@@ -0,0 +1,53 @@
+<!--
+  This template is not mandatory.
+  It simply serves to provide a guide to allow a better review of pull requests.
+-->
+
+## What is in this PR?
```

Comment:
I'm sorry, but now that I've given this some thoughts, I am against that section too: we already have that information with the `changelog:*` labels. 
So not only does this section duplicates that information, but our labels have been set using standard changelog categories, and are used by our release tool to generate a standardized version's changelog automatically.

Reply:
Following our discussion: this section can be replaced by a comment telling that putting a `changelog` label is strongly encouraged (PRs without such a label will be shown in the `Others` section of generated changelogs)